Neos.ContentRepository.Core/Classes/Projection/CheckpointStorageStatus.php

```diff
@@ -7,23 +7,34 @@
  */
 final readonly class CheckpointStorageStatus
 {
+    /**
+     * @param non-empty-string $details
```

Comment:
That should be removed IMO

Reply:
the reason its there to enforce that there is a message for error cases, which should be statically enforced.
Otherwise we might lazy and forget it :D